README.md

```diff
@@ -3,7 +3,11 @@
 Hunger Games is a series of mini-apps that let users contribute data to Open Food Facts, in a rather fun way using React.
 You are very welcome to write the game you want.
 
-[![Open Food Facts](https://static.openfoodfacts.org/images/misc/openfoodfacts-logo-en.svg)](https://world.openfoodfacts.org/)
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://static.openfoodfacts.org/images/logos/off-logo-horizontal-dark.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://static.openfoodfacts.org/images/logos/off-logo-horizontal-light.svg">
+  <img height="100" src="https://static.openfoodfacts.org/images/logos/off-logo-horizontal-light.svg">
+</picture>
```

Comment:
If you want github support dark mode natively with `#gh-dark-mode-only` and `#gh-light-mode-only`

```suggestion
![Open Food Facts](https://static.openfoodfacts.org/images/logos/off-logo-horizontal-dark.svg#gh-dark-mode-only)
![Open Food Facts](https://static.openfoodfacts.org/images/logos/off-logo-horizontal-light.svg#gh-light-mode-only)
```